hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/HttpBackendResolver.java

```diff
@@ -66,11 +66,10 @@
       if (httpOperation != null) {
         enrichedAttributes.put(BACKEND_OPERATION_ATTR, AttributeValueCreator.create(httpOperation));
       }
-      Optional<String> httpDestination = HttpSemanticConventionUtils.getHttpTarget(event);
-      httpDestination.ifPresent(
-          destination ->
-              enrichedAttributes.put(
-                  BACKEND_DESTINATION_ATTR, AttributeValueCreator.create(destination)));
+
+      if (StringUtils.isNotEmpty(path)) {
```

Comment:
what if http target attribute is present in the tags?

Reply:
Added details in descripiton. Its is covered by considering `http.request.path`, `http.path`, `http.target` and extracting from `http.url`